### ISSUE TITLE
Split only one type of object

### ISSUE THREAD

**Issue by zqnnn** [CSAILVision/semantic-segmentation-pytorch#148]
I just want to segment only one type of object, such as road, what should I do? Thank you~

**Comment by Michriko** [comment]
There are multiple ways to do so. 
I think the best way would be to train a two class segmentation network. Class 1 is for road, class 2 is for everything except the road. 
You also have to prepare your trainingdata or the part in the code, where the training data is loaded. modify all entrys in the label. Entries, that are road become a 1, all the other ones become 0. 

That way you can train a two class segmentation network. 

Another way would be to use a working network and modify its output after inference.
